quarkus-extension/engine/runtime/src/main/java/org/camunda/bpm/quarkus/engine/extension/impl/ProcessEngineStartEvent.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.quarkus.engine.extension.impl;
+
+/**
+ * A CDI Event class fired when the Camunda process engine is bootstrapped successfully.
+ */
+public class ProcessEngineStartEvent {
```

Comment:
Two things I have in mind about the name:
* Quarkus uses the following name `StartupEvent`: should we adopt this naming convention?
* Not sure if the explicit term `Process` is needed; the engine is much more than just a Process Engine (cf. embedded DMN Engine)
  * At other places in this Quarkus Extension codebase we just use the term `engine`, e.g., in the package name or the maven artifact id
  * We use `QuarkusProcessEngineConfiguration`, however, here we extend the `ProcessEngineConfiguration` so it makes somehow sense to keep the `Process` term

Reply:
* Regarding adopting the same naming convention as Quarkus, definitely.
* I am not sure if we should drop the `Process` term. The object that we fire the event for is still an instance of `ProcessEngine`. Keep in mind that this code will be used with other Quarkus extensions as well. It might be better if the naming is clearer.

What do you think about `ProcessEngineStartupEvent` or `CamundaEngineStartupEvent`?

Reply:
I agree with you that the event is emitted when `ProcessEngine` is created, and maybe I even inject this object and call some APIs. Under this assumption, I would vote for `ProcessEngineStartupEvent`.